.github/workflows/build.yml

```diff
@@ -0,0 +1,127 @@
+name: Build
+
+on:
+  workflow_dispatch:
```

Comment:
I don't want every push to trigger a build, so I choose to let users manually trigger it. See: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#on